packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -72,14 +92,16 @@ export const contract = async (
     makeOpenPortfolioInvitation() {
       trace('makeOpenPortfolioInvitation');
       return zcf.makeInvitation(
-        (seat, offerArgs) =>
-          openPortfolio(
+        (seat, offerArgs: OfferArgsShapes) => {
```

Comment:
You're using `OfferArgsShapes` to name a type. types disappear at runtime. Shapes do not. Shapes are runtime values. They're patterns.